back/app/services/notification_service.rb

```diff
@@ -6,31 +6,21 @@ class NotificationService
     Notifications::CommentDeletedByAdmin,
     Notifications::CommentMarkedAsSpam,
     Notifications::CommentOnIdeaYouFollow,
-    Notifications::CommentOnInitiativeYouFollow,
```

Comment:
We will also need to delete existing notifications on existing platforms.

Reply:
Discussed with Sebi, and these will be removed automatically via rake task run on CI, which migration step happens (which happens every deploy to staging or production)